src/experiments.js

```diff
@@ -66,6 +66,15 @@ export function isCanary(win) {
   return !!(win.AMP_CONFIG && win.AMP_CONFIG.canary);
 }
 
+/**
+ * Returns runtime type, e.g., canary, control, or production.
+ * @param {!Window} win
+ * @return {string}
+ */
+export function getRuntimeType(win) {
```

Comment:
I'm adding @erwinmombay and @rsimha-amp for review since they own this overall. My recommendation, however, is to call it something other than "runtime type" - we already have that concept for "single doc" vs "shadow mode" vs "in-a-box". Maybe "binary type"? Or "coarse version"? Or???

Reply:
I'm leaning toward getBinaryType unless there are other suggestions.